### PR TITLE
Repro 2

### PR DESCRIPTION
**Pull request by Hgherzog**

### CODE REVIEW

File: helios/evals/datasets/normalize.py
Comment (Hgherzog) [inline helios/evals/datasets/normalize.py:1]: maybe this changes evla if there is a bug

File: helios/evals/linear_probe.py [hunk 1]
```diff
@@ -127,9 +126,19 @@ def train_and_eval_probe(
     if train_embeddings.shape[-1] != test_embeddings.shape[-1]:
         raise ValueError("Embedding dims don't match.")
     in_features = train_embeddings.shape[-1]
-
+    output_pixels_per_side_of_patch = None
     if config.task_type == TaskType.SEGMENTATION:
-        logits_per_patch = int(config.num_classes * patch_size * patch_size)
+        assert config.height_width is not None, (
```
Comment (Hgherzog) [inline helios/evals/linear_probe.py:131]: This touches the pastis eval but I think it is right
